R/pkg/R/mllib.R

```diff
@@ -694,8 +694,8 @@ setMethod("predict", signature(object = "KMeansModel"),
 #' }
 #' @note spark.mlp since 2.1.0
 setMethod("spark.mlp", signature(data = "SparkDataFrame"),
-          function(data, blockSize = 128, layers = c(3, 5, 2), solver = "l-bfgs", maxIter = 100,
-                   tol = 0.5, stepSize = 1, seed = 1) {
+          function(data, blockSize = 128, layers, solver = "l-bfgs", maxIter = 100,
```

Comment:
@felixcheung but `layers = c()` the `c()` is an invalid value for `layers` parameter.... so I think it is better not to specify the default value for `layers` so user must specify this parameter.